### PR TITLE
[Codegen] Clean Up & Fix Input Variable Naming & De-Duplication

### PR DESCRIPTION
**Pull request by noanflaherty**
Fixes input name generation and de-duplication, following patterns establish by node name de-duping.

### CODE REVIEW

File: ee/codegen/src/__test__/__snapshots__/inputs.test.ts.snap [hunk 1]
```diff
@@ -60,8 +60,8 @@ exports[`Inputs > write > should generate correct code when Inputs has variables
 
 
 class Inputs(BaseInputs):
-    var_1: str
-    var_2: str
+    input_: str
```
Comment (NgoHarrison) [inline ee/codegen/src/__test__/__snapshots__/inputs.test.ts.snap:63]: [q] is this intentional naming?

Comment (noanflaherty) [reply ee/codegen/src/__test__/__snapshots__/inputs.test.ts.snap:63]: Yeah – I wanted to make the default name "input", but that's a reserved keyword in python, so a common approach is to add a trailing "_"

Maybe `Inputs.input_` is bad though. Open to alternative naming suggestion, but I had a slight preference for this over `var`

Comment (NgoHarrison) [reply ee/codegen/src/__test__/__snapshots__/inputs.test.ts.snap:63]: Ah I see. Yup I like input_ over the other one as well

**Comment by NgoHarrison** [comment]
✅

**Review by NgoHarrison — COMMENTED** [review]
✅

**Pull request merged**